src/bigint.rs

```diff
@@ -538,7 +540,7 @@ impl IntDigits for BigInt {
         self.data.digits()
     }
     #[inline]
-    fn digits_mut(&mut self) -> &mut Vec<BigDigit> {
+    fn digits_mut(&mut self) -> &mut SmallVec<[BigDigit; BigUint::INLINED]> {
```

Comment:
I suggest adding an aliased `type BigVec = SmallVec<...>` in one place, so the particular type details aren't duplicated everywhere. The exact name is not important to me, if you'd prefer `BigData` or something else, as long as it's clear.